src/onegov/org/templates/macros.pt

```diff
@@ -562,7 +562,8 @@
                     <h2 i18n:translate>People</h2>
                     <ul class="panel-links" data-sortable data-sortable-url="${layout.move_person_url_template}">
                         <li tal:repeat="person people" data-sortable-id="${request.is_manager and person.id}" class="${repeat.person.index > 2 and 'hidden hideable' or ''}">
-                            <a href="${request.link(person)}">${person.title}</a>
+                            <a tal:condition="western_ordered" href="${request.link(person)}">${person.first_name} ${person.last_name}</a>
+                            <a tal:condition="not western_ordered" href="${request.link(person)}">${person.title}</a>
```

Comment:
This seems pretty fragile, requiring every view to pass this in. The point of extensions is, that you can rely on every model that uses the extension to have the required data accessible from the same attribute, so you don't have to do quite as much plumbing, you should be able to just do `page.show_western_name_order` here, please don't add additional template data that isn't actually required, it makes template rendering more error prone.